### PR TITLE
Added support for Bitbucket urls

### PR DESCRIPTION
**Pull request by julianbraasch**
Adresses #29

### CODE REVIEW

**Comment by vanlooverenkoen** [comment]
Fix merge conflicts. & restart travis for testing.

**Review by vanlooverenkoen — DISMISSED** [review]

**Comment by vanlooverenkoen** [comment]
@julianbraasch can you fix these merge conflicts?

File: bin/src/command/generate_command.dart [hunk 1]
```diff
@@ -75,7 +75,7 @@ class GenerateCommand {
       ..writeln('  LicenseUtil._();')
       ..writeln()
       ..writeln('  static List<License> getLicenses() {')
-      ..writeln('    return [');
```
[the remark this answers is not in the document]
Comment (ikbendewilliam) [reply bin/src/command/generate_command.dart:78]: I don't mind this, you can still expand it like this:
```dart
license = [
...LicenseUtil.getLicenses(),
someOtherLicense,
],
```

[the remark this answers is not in the document]
Comment (vanlooverenkoen) [reply bin/src/command/generate_command.dart:78]: Yes indeed. alright. just checking

**Comment by vanlooverenkoen** [comment]
@julianbraasch your formatting fails on travis

**Comment by julianbraasch** [comment]
@vanlooverenkoen Fixed. That formattersure is very picky 😄
I also removed that const keyword from the list again because it broke other tests.

**Comment by vanlooverenkoen** [comment]
It is using the dart formatter to keep hogh scores on pub.dev

**Review by vanlooverenkoen — APPROVED** [review]

**Pull request merged**